src/core/session.cpp

```diff
@@ -594,6 +594,30 @@ class Session
         }
 
         mod_wrapper.set_mod(next_state, mod_pack);
+
+        if (this->ini.get<cfg::context::banner_activate>())
+        {
+            this->ini.set<cfg::context::banner_activate>(false);
```

Comment:
based on the content of `cfg::context::banner_message`, `banner_activate` is not needed anymore.